var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -7,28 +7,41 @@
 
 
 class PyFenicsDolfinx(PythonPackage):
-    """Python interface library to Next generation FEniCS problem solving
+    """Python interface to the next generation FEniCS problem solving
     environment"""
 
     homepage = "https://github.com/FEniCS/dolfinx"
     url = "https://github.com/FEniCS/dolfinx/archive/v0.1.0.tar.gz"
     git = "https://github.com/FEniCS/dolfinx.git"
-    maintainers = ["chrisrichardson", "garth-wells", "nate-sime"]
+    maintainers = ["chrisrichardson", "garth-wells", "nate-sime", "jhale"]
 
     version("main", branch="main")
+    version("0.5.1", sha256="a570e3f6ed8e7c570e7e61d0e6fd44fa9dad2c5f8f1f48a6dc9ad22bacfbc973")
     version("0.5.0", sha256="503c70c01a44d1ffe48e052ca987693a49f8d201877652cabbe2a44eb3b7c040")
     version("0.4.1", sha256="68dcf29a26c750fcea5e02d8d58411e3b054313c3bf6fcbc1d0f08dd2851117f")
-    version("0.3.0", sha256="4857d0fcb44a4e9bf9eb298ba5377abdee17a7ad0327448bdd06cce73d109bed")
-    version("0.2.0", sha256="4c9b5a5c7ef33882c99299c9b4d98469fb7aa470a37a91bc5be3bb2fc5b863a4")
-    version("0.1.0", sha256="0269379769b5b6d4d1864ded64402ecaea08054c2a5793c8685ea15a59af5e33")
+    version(
+        "0.3.0",
+        sha256="4857d0fcb44a4e9bf9eb298ba5377abdee17a7ad0327448bdd06cce73d109bed",
+        deprecated=True,
+    )
+    version(
+        "0.2.0",
+        sha256="4c9b5a5c7ef33882c99299c9b4d98469fb7aa470a37a91bc5be3bb2fc5b863a4",
```

Comment:
This also appears to be incorrect and should be:

```suggestion
        sha256="e6462fc3b9653d335c28096f9b0393f966c55a64c77ed64cc6c949406cb1f2c3",
```